pkg/segment/segment.go

```diff
@@ -300,3 +303,11 @@ func sanitizeExec(errString string) string {
 	errString = pattern.ReplaceAllString(errString, fmt.Sprintf("exec command %s", Sanitizer))
 	return errString
 }
+
+func GetDebugTelemetry() string {
+	return os.Getenv(DebugTelemetryFileEnv)
+}
+
+func SetDebugTelemetry(value string) error {
+	return os.Setenv(DebugTelemetryFileEnv, value)
+}
```

Comment:
I don't think we need this function here in the odo source. If you need it for the tests, please move the declaration of the function in the tests packages (or use os.SetEnv directly)

Reply:
ok